cruise/CruiseInformation.csv

```diff
@@ -256,5 +256,6 @@ EK20210714,Elakha,2021-07-14T15:12:00,2021-07-14T23:48:00,Endurance Glider opera
 EK20210715,Elakha,2021-07-15T15:30:00,2021-07-15T21:30:00,Endurance ce02shsp r00021 d00022 and O2 samples ce01 ce02
 SKQ202111S,Sikuliaq,2021-07-18T00:00:00,2021-08-02T23:59:00,Papa 8
 WJ20210721,Warren Jr.,2021-07-21T21:00:00,2021-07-23T09:00:00,Pioneer glider cp_379 recovery
+EK20210730,Elakha,2021-07-30T15:39:00,2021-07-30T18:37:00,Endurance glider deployment ce_917-00001

```

Comment:
looks good (assuming the times are correct - no way to verify)

Reply:
I verify they are correct.  Excerpt from my cruise log (first time PDT second UTC):
08:39/15:39 left the dock
...
11:02/18:02 arrived back at the dock

Reply:
Looks good - thx!

Reply:
Looks fine.